lib/core/src/display/shape/text/text_field.rs

```diff
@@ -143,21 +160,21 @@ shared! { TextField
 
         /// Jump active cursor to point on the screen.
         pub fn jump_cursor(&mut self, point:Vector2<f32>, selecting:bool) {
+            let point_on_text  = self.relative_position(point);
             let content        = &mut self.content;
-            let text_position  = self.rendered.display_object.global_position();
-            let point_on_text  = point - text_position.xy();
-            let mut navigation = CursorNavigation {content,selecting};
+            let jumping_words  = false;
+            let mut navigation = CursorNavigation {content,selecting,jumping_words};
```

Comment:
I would prefer to use `CursorNavigation {content,selecting,..default()}` in such a situation, as it looks like a builder pattern to me and it will grew in the future (if thats possible ofc).